library/Zend/Filter/Bytes.php

```diff
@@ -0,0 +1,245 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Filter;
+
+use Zend\Filter\Exception\InvalidArgumentException;
+
+final class Bytes extends AbstractFilter
```

Comment:
The filter name is a bit ambiguous: I was thinking of a binary string filter when I read the topic title, but now I understand that it is about measurement units. Maybe it's a generic measurement unit filter?